sandbox/namespace_linux.go

```diff
@@ -254,6 +255,37 @@ func (n *networkNamespace) SetGatewayIPv6(gw net.IP) error {
 	return err
 }
 
+func (n *networkNamespace) AddStaticRoute(r *types.StaticRoute) error {
+	n.Lock()
+	defer n.Unlock()
```

Comment:
In libnetwork we never lock the code only the data. Please move the `Lock/Unlock` inside the `if` clause